ocaml/xapi-idl/xen/xenops_interface.ml

```diff
@@ -718,6 +718,11 @@ module XenopsAPI (R : RPC) = struct
           ~description:["when true, verify remote server certificate"]
           Types.bool
       in
+      let localhost_migration =
```

Comment:
Why does this need a parameter? Can this not been inferred? And can this not lead to inconsistencies? Are the semantics of this not actually something else: don't balloon down?

Reply:
I didn't find any other way to infer this in xenopsd, maybe I missed something. Currently the semantics are "don't baloon down" but there are potentially other optimizations to make for same-host migration so I made it more general.